bootstrap/kubeadm/internal/locking/control_plane_init_mutex_test.go

```diff
@@ -110,6 +110,8 @@ func TestControlPlaneInitMutex_Lock(t *testing.T) {
 	for _, tc := range tests {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
+			gs := NewWithT(t)
```

Comment:
is there a reason for using `gs` here instead of `g`?

Reply:
It was a suggestion of mine as there is a `g` in the outer scope that this was then shadowing, thought it might make it more obvious to use a different name for the variable

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/2799#pullrequestreview-382791080

Reply:
do we need the parent `g`?

Reply:
In the cases where `gs` is used, yes, the parent `g` is required

Reply:
yes, for the reason that Joel explained